compiler_opt/es/regalloc_trace/regalloc_trace_worker.py

```diff
@@ -27,15 +27,22 @@
 import concurrent.futures
 import tempfile
 import shutil
+from typing import Any
 
 import gin
+import tensorflow as tf
 
 from compiler_opt.rl import corpus
 from compiler_opt.distributed import worker
 from compiler_opt.rl import policy_saver
 from compiler_opt.es import policy_utils
 
 
+def _make_dirs_and_copy(old_path: str, new_path: str):
```

Comment:
add `_file`. And `old_file_path` and `new_file_path`. So noone believes later this applies to dirs.